routes/studentRoutes.js

```diff
@@ -4,6 +4,7 @@ const { markAsCompleted } = require('../helpers/answerExercises');
 const requireLogin = require('../middlewares/requireLogin');
 const mongoose = require('mongoose');
 const { encrypt, compare } = require('../helpers/password');
+const { findTop100Students } = require('../helpers/leaderboard');
```

Comment:
should you be able to see yourself in comparison to the top 100 on the leaderboard?